src/main/scala/de/m7w3/signal/ApplicationContext.scala

```diff
@@ -41,7 +41,7 @@ case class ApplicationContext(config: Config.SignalDesktopConfig) {
   }
 
   def tryLoadExistingStore(password: String): Try[SignalDesktopProtocolStore] = {
-    databaseContext.dbActionRunner(password, onlyIfExists = true)
+    databaseContext.dbActionRunner(password, onlyIfExists = false)
```

Comment:
@mfelsche had to change it to avoid throwing an DatabaseDoesNotExistException [here](https://github.com/ayoub-benali/signal-desktop-client/blob/master/src/main/scala/de/m7w3/signal/ApplicationContext.scala#L83)